common-content/modules/ROOT/partials/cypher-workflow.adoc

```diff
@@ -457,24 +457,24 @@ All types can be potentially found in the result, although not all types can be
 [options="header", cols=",,"]
 |===
 | Cypher Type | Parameter | Result
-| `null`* | {check-mark} | {check-mark}
-| `List`  | {check-mark} | {check-mark}
-| `Map`   | {check-mark} | {check-mark}
-| `Boolean` | {check-mark} | {check-mark}
-| `Integer` | {check-mark} | {check-mark}
-| `Float` | {check-mark} | {check-mark}
-| `String` | {check-mark} | {check-mark}
+| `NULL`* | {check-mark} | {check-mark}
```

Comment:
```suggestion
| `null`* | {check-mark} | {check-mark}
```

We should refer to `null` always in the same way. Applies to all instances later below :)

Reply:
I had a chat with @Hunterness and as far as I understand, we should refer to the Cypher type as `NULL`.

Reply:
> I had a chat with @Hunterness and as far as I understand, we should refer to the Cypher type as NULL.

Fine by me, but why does line 147 makes an exception? :thinking:

Reply:
I assume you mean the line `147` mentioned here: https://github.com/neo4j/docs-drivers/pull/400#discussion_r1236792854

It refers to `null` as a value, not the type. @Hunterness please correct me if I am wrong.